src/expand.rs

```diff
@@ -166,8 +178,10 @@ impl Instantiator {
 impl VisitMut for Instantiator {
     fn visit_item_mod_mut(&mut self, item: &mut ItemMod) {
         debug_assert_ne!(self.depth, 0);
-        match InstArguments::try_extract(item) {
-            Ok(Some(args)) => {
+        match InstArguments::try_extract(item)
+            .and_then(|args| TestAttrs::try_extract(item).map(|attrs| (args, attrs)))
```

Comment:
Tiny nit: I'd find the more imperative syntax a bit more readable 🙂:

```suggestion
            .and_then(|args| Ok((args, TestAttrs::try_extract(item)?)))
```